syntax/parser_test.go

```diff
@@ -156,10 +156,20 @@ func TestParseBats(t *testing.T) {
 }
 
 func TestMain(m *testing.M) {
-	// Set the locale to computer-friendly English and UTF-8, C.UTF-8,
-	// which started shipping with glibc 2.35 in February 2022.
-	os.Setenv("LANGUAGE", "C.UTF-8")
-	os.Setenv("LC_ALL", "C.UTF-8")
+	// Set the locale to computer-friendly English and UTF-8.
+	// Some systems like Arch miss C.UTF8, so fall back to the US English locale.
+	//
+	// TODO: glibc 2.35 was released with C.UTF-8 in February 2022.
+	// This locale is now becoming a standard, so remove the workaround in 2023.
```

Comment:
this TODO is wrong now then, as we're in 2025. it should probably mention that macOS still does not ship the C locale.

Reply:
I just updated the comment to mention macOS.